server/http.go

```diff
@@ -358,7 +358,7 @@ func handleAuthFunc(srv OIDCServer, idpcs []connector.Connector, tpl *template.T
 			if ok {
 				q := url.Values{}
 				q.Set("code", key)
-				ru := httpPathRegister + "?" + q.Encode()
+				ru := strings.TrimRight(baseURL, "/") + "/" + strings.TrimLeft(httpPathRegister, "/") + "?" + q.Encode()
```

Comment:
Could you maybe use `path.Join` or `url.Parse` instead? It should remove some of the need to trim, but is also more likely work correctly with joining paths together.